src/Core/src/Core/IEntry.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Microsoft.Maui
 {
-	public interface IEntry : IView, IText
+	public interface IEntry : IView, IText, ITextInput
```

Comment:
I think `ITextInput` has a bunch of props, so maybe we might have to delete them all for now except the placeholder?

Reply:
I would leave them for now, because other handlers are coming in a few days :)

Reply:
But will it compile without being implemented in the stubs?

Also, we do want to remove the color property entirely as we will use a brush or something.

Reply:
With the last commit, I implemented them in the Entry stub.

"Also, we do want to remove the color property entirely as we will use a brush or something." - Ok, then I will do based on suggestions when we figure out what to do with ITextInput :)

Reply:
Let's do this in a new PR. Then we can merge this sooner :)

Reply:
I removed the usage of ITextInput, when we have a clear clue what to do with that interface we can add it later, now currently the Placeholder is only property (not coming from the interface). Hope now it is ready for merge 😄